lib/api/objectPutPart.js

```diff
@@ -238,12 +243,20 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                     }
                     return next(null, destinationBucket,
                         objectLocationConstraint, cipherBundle,
-                        partKey, prevObjectSize, oldLocations);
+                        partKey, prevObjectSize, oldLocations, partInfo);
                 });
         },
         // Store in data backend.
         (destinationBucket, objectLocationConstraint, cipherBundle,
-            partKey, prevObjectSize, oldLocations, next) => {
+        partKey, prevObjectSize, oldLocations, partInfo, next) => {
+            // NOTE: set oldLocations to null so we do not batchDelete for now
```

Comment:
As discussed, it sounds like if you put part 1 again and it has 5 subparts, it will overwrite the prior say 5 subparts.  We should test that -- make sure if you put a block with the same name again, it does not error and that the committed final blob has the new data. If there were 6 subparts before, the last one will be cleaned up when you commit a block list without it.